dist/htmlminifier.js

```diff
@@ -1503,7 +1514,22 @@
                 });
               }
             }
+            if (prevTag && inlineTextTags(prevTag.charAt(0) === '/' ? prevTag.substr(1) : prevTag)) {
+              text = collapseWhitespace(text, options, /(?:^| )$/.test(currentChars));
+            }
             text = prevTag || nextTag ? collapseWhitespaceSmart(text, prevTag, nextTag, options) : trimWhitespace(text);
+            if (!text && / $/.test(currentChars) && prevTag && prevTag.charAt(0) === '/') {
+              for (var index = buffer.length - 2, endTag = prevTag.substr(1); index >= 0 && _canTrimWhitespace(endTag); index--) {
```

Comment:
I thought of using `slice`, but then looking up and saw we use `substr` everywhere else :sweat_smile: 

Do you want me to replace all the usages?